dialogue-annotations-example/src/test/java/com/palantir/myservice/example/MyServiceIntegrationTest.java

```diff
@@ -316,6 +317,17 @@ void testMultiparams() {
                 new MyCustomParamType("var3"));
     }
 
+    @Test
+    void testMultiplePathParams() {
+        UUID first = UUID.randomUUID();
+        UUID second = UUID.randomUUID();
+        undertowHandler = exchange -> {
+            exchange.assertMethod(HttpMethod.GET);
+            exchange.assertPath("/multipath/" + first + '/' + second);
+        };
+        myServiceDialogue.multiplePathSegments(ImmutableList.of(first, second));
```

Comment:
Do we want a test that includes a `/` char to guard against future refactors messing up assumptions?

Reply:
(As in, to ensure it's properly escaped) Or do we assume this is already covered by existing tests?

Reply:
No harm in adding one, however that's a separate concern from the code impacted by this change. This works nicely because we take in a collection (boundaries defined in code) and simply apply them as segments to the uri using the same method we've always used. The function which applies individual segments is responsible for encoding characters in the value.